packages/jellyfish-transaction-builder/src/txn/txn_builder_vault.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  OP_CODES, Script, TransactionSegWit,
+  CreateVault,
+  UpdateVault,
+  DepositToVault,
+  WithdrawFromVault,
+  CloseVault,
+  PlaceAuctionBid
+} from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { BigNumber } from '@defichain/jellyfish-json'
+
+export class TxnBuilderVault extends P2WPKHTxnBuilder {
+  /**
+   * Creates vault transaction.
+   *
+   * @param {CreateVault} createVault txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+
```

Comment:
```suggestion
   */
```

Reply:
hmm... there's still contain extra break line below :thinking:

Reply:
I am thinking could the extra indentation and break-line were caused by vs-code plugins?

Reply:
hmm... I need to check again. Things seems quite weird.

Reply:
Removed the whitespaces, seems like this might be my fault instead of the formatter.